README.md

```diff
@@ -190,6 +190,10 @@ Customize the font family to be used for the title.
 
 Customize the size of the font to be used for the title.
 
+#### `titleFontWeight`
```

Comment:
If it is not available on Android, please add the `(iOS only)` in all READMEs and other files with this.